routes/api.js

```diff
@@ -90,4 +93,35 @@ api.delete('/friendships/user/:secret', async function (req, res) {
     })
 })
 
+// Route for merging accounts
+api.put('/user/:secret', async function (req, res) {
+  const { secret } = req.params
+  const { mergeUrl } = req.body
+  const mergeSecret = mergeUrl.split('/user/')[1]
+
+  // Checks for invalid url and prevents merging into itself
+  if (mergeSecret === undefined || secret === mergeSecret) {
+    return res.status(400).send('Bad url')
+  }
+
+  // Checks for existing account
+  const userIds = await UserService.getUserIdsFromSecrets([mergeSecret, secret])
+  if (userIds[mergeSecret] === null || userIds[secret] === null) {
+    console.log(userIds[mergeSecret])
+    console.log(userIds[secret])
+    return res.status(404).send()
+  }
+  UserMergeService.mergeAccounts(userIds[mergeSecret], userIds[secret])
+    .then(() => {
+      res.status(200).send()
+    })
+    .catch(err => {
+      console.log(err)
+      if (err.message === 'accountHasBeenMerged') {
```

Comment:
[nit] constants?

Reply:
constants?

Reply:
Make `'accountHasBeenMerged'` and a constant

Reply:
Should I make a new constants file in the services folder for errors and such?